MapboxNavigation/NavigationMapView.swift

```diff
@@ -12,6 +14,11 @@ open class NavigationMapView: MGLMapView {
     let routeLayerIdentifier = "routeLayer"
     let routeLayerCasingIdentifier = "routeLayerCasing"
     
+    let TrafficSevere = UIColor(red:0.54, green:0.06, blue:0.22, alpha:1.0)
+    let TrafficHeavy = UIColor(red:0.91, green:0.20, blue:0.25, alpha:1.0)
+    let TrafficModerate = UIColor(red:0.95, green:0.65, blue:0.31, alpha:1.0)
+    let TrafficLowUnknown = UIColor(red:0.00, green:0.70, blue:0.99, alpha:1.0)
```

Comment:
`TrafficLowUnknown` should somehow correspond to the route line color, right? What if the developer wants to change the route line’s color?